app.js

```diff
@@ -0,0 +1,56 @@
+const config = require('./config');
+require('express-async-errors');
+
+// express
+const express = require('express');
+const app = express();
+
+// packages
+const morgan = require('morgan');
+const cookieParser = require('cookie-parser');
+const rateLimiter = require('express-rate-limit');
+const helmet = require('helmet');
+const xss = require('xss-clean');
+const cors = require('cors');
+const mongoSanitize = require('express-mongo-sanitize');
+
+// database
+const connectDB = require('./db/connect');
+
+// routers
+const authRouter = require('./routes/authRoutes');
+
+// middleware
+
+app.set('trust proxy', 1);
```

Comment:
I'm not really sure we need this? Why do you think we'll be using proxies?

Reply:
I remove it. :)